### PR TITLE
[Dialogs] Add snapshot test for actions layout issues.

### PR DESCRIPTION
**Pull request by galiak11**
# Description
Adding a snapshot test to verify correct layout of actions in edge cases found in issue: #8434.

The 2 issues found in #8434 are corrected in PRs: #9878 and #9877.

Issue 1:  Content size is calculated based on previous layout calculations. When the actions view grows in size, for instance, when actions switch from lowercase to uppoercase, the layout size doesn't grow, and retain the vertical orientation instead of horizontal orientation.

Issue 2: Action width is calculated without the margins. When the actions are the widest in the dialog, it is sized too narrowly, truncating the largest action (using ellipsis).

## Screenshot:
image

## Issues
b/137070719, cl/299739784

### CODE REVIEW

File: components/Dialogs/tests/snapshot/MDCAlertControllerActionsTests.m [hunk 1]
```diff
@@ -407,11 +450,22 @@ - (void)addFirstLongActionWithEmphasis:(MDCActionEmphasis)emphasis {
...
+                                                         emphasis:emphasis
+                                                          handler:nil]];
+}
+
+- (void)addLowEmphasisAction:(NSString *)action {
+  [self.alertController addAction:[MDCAlertAction actionWithTitle:action
+                                                         emphasis:MDCActionEmphasisLow
+                                                          handler:nil]];
+}
+
```
Comment (jverkoey) [inline components/Dialogs/tests/snapshot/MDCAlertControllerActionsTests.m:450]: Please pull the refactoring out to a separate PR and then land the new snapshot test independently.

**Review by jverkoey — CHANGES_REQUESTED** [review]

**Comment by jverkoey** [comment]
To confirm: this new snapshot test is supposed to be revealing a layout bug?

**Comment by jverkoey** [comment]
If so, please update the PR description accordingly.

**Comment by galiak11** [comment]
It reveals 2 layout bugs - added the info in the description.

**Comment by jverkoey** [comment]
Please remove the other PR description snippets from this PR description; it reads like this PR is fixing the Issues presented. Please focus instead on explaining what Issues this PR is revealing.

**Comment by galiak11** [comment]
Please review fixed decsription.

**Review by jverkoey — APPROVED** [review]
